.changeset/hot-pumpkins-clean.md

```diff
@@ -0,0 +1,26 @@
+---
+'@makeswift/controls': minor
+'@makeswift/runtime': minor
+---
+
+## Breaking change
+
+### Updated `Shape` control
+
+The `Shape` control has been updated with two new params `label` and `layout`.
+
+- `label?: string = "Group"`
+  - Text for the panel label in the Makeswift builder.
+  - This `label` defaults to "Group."
+- `layout?: 'Shape.Layout.Inline | Shape.Layout.Popover = Shape.Layout.Popover`
+  - The configuration for how panels are rendered in the right sidebar.
```

Comment:
```suggestion
  - The configuration for how panels are rendered in the Properties Sidebar.
```

We should use the same naming we use in our docs

Reply:
💯. I agree.  I couldn't find [this page](https://docs.makeswift.com/product/builder-basics#the-basics) at the time, when I was searching there were quite a few "right sidebar" usages, but I like this change.

![CleanShot 2024-12-20 at 14 45 32@2x](https://github.com/user-attachments/assets/95c00a42-7268-4268-b158-06800cff3662)

Reply:
Ping @jamesqquick. It might be good to update these "right sidebar" instances to say Properties sidebar.